### ISSUE TITLE
Create Attribute for nf_conntrack_max

### ISSUE THREAD

**Issue by jcannava** [rcbops/chef-cookbooks#817]
Under high traffic instance log  messages appear to be repeating related to the nf_conntrack table being full.

Feb 27 21:17:40 548119-Controller01 kernel: [3544271.165123] nf_conntrack: table full, dropping packet
Feb 27 21:17:40 548119-Controller01 kernel: [3544271.197377] nf_conntrack: table full, dropping packet
Feb 27 21:17:40 548119-Controller01 kernel: [3544271.199579] nf_conntrack: table full, dropping packet
Feb 27 21:17:41 548119-Controller01 kernel: [3544271.724614] nf_conntrack: table full, dropping packet
Feb 27 21:17:41 548119-Controller01 kernel: [3544271.803820] nf_conntrack: table full, dropping packet

In order to clear this error increased /proc/sys/net/netfilter/nf_conntrack_max to 256000. Suggest creating an attribute so that we can change this across clusters that have high traffic patterns.

**Comment by cloudnull** [comment]
done. in v4.2.2RC and master

**Comment by cjoelrun** [comment]
sysctl lwrp requires string not integer

**Comment by claco** [comment]
Cherry picked into 4.2.2rc:  https://github.com/rcbops-cookbooks/osops-utils/pull/170

**Comment by xeregin** [comment]
I verified that this is indeed working as intended by modifying the environment to include an nf_conntrack_max value of 256000 on the chef server. I then ran chef client on controller1 and verified that the nf_conntrack_max value had been successfully modified:
root@ian-controller:~# cat /proc/sys/net/netfilter/nf_conntrack_max
256000

**Comment by xeregin** [comment]
Is it valid for this to fail when assigned in the environment? This is what happens on the compute node.

```
================================================================================
Error executing action `run` on resource 'execute[sysctl-p]'
================================================================================


Mixlib::ShellOut::ShellCommandFailed
------------------------------------
Expected process to exit with [0], but received '255'
---- Begin output of sysctl -p /etc/sysctl.d/40-net.nf_conntrack_max.conf ----
STDOUT:
STDERR: error: "net.nf_conntrack_max" is an unknown key
---- End output of sysctl -p /etc/sysctl.d/40-net.nf_conntrack_max.conf ----
Ran sysctl -p /etc/sysctl.d/40-net.nf_conntrack_max.conf returned 255


Resource Declaration:
---------------------
# In /var/chef/cache/cookbooks/sysctl/providers/default.rb

 24:   execute "sysctl-p" do
 25:     command "sysctl -p #{fullname}"
 26:     action :nothing
 27:   end
 28:
```

**Comment by claco** [comment]
What's in `/etc/sysctl.d/40-net.nf_conntrack_max.conf`?

Or better yet, the diff between that file on compute compared to that file on controller.

**Comment by xeregin** [comment]
Alright, so it seems that it was an anomaly, because I was unable to reproduce the issue on a fresh build. I changed the environment to contain an nf_conntrack_max value of 256000 and ran chef-client on the controller (verifying values before and after). I then tried running chef-client on the compute node and it worked fine (the compute node does not have the nf_conntrack_max attribute which I believe is expected). I then ran chef-client on controller2 (verifying values before and after). chef-client still ran successfully on compute after everything had been modified.

TL;DR
It works!
